mathics/doc/latex_doc.py

```diff
@@ -523,7 +521,9 @@ def latex(self, doc_data: dict) -> str:
             output_for_key = get_results_by_test(self.test, self.key, doc_data)
         text = f"%% Test {'/'.join((str(x) for x in self.key))}\n"
         text += "\\begin{testcase}\n"
-        text += "\\test{%s}\n" % escape_latex_code(self.test)
+        test_str = self.test
+        # TODO: replace nonascii characters in test_str
+        text += "\\test{%s}\n" % escape_latex_code(test_str)
```

Comment:
Here I copy the attribute into a variable, because eventually we would like to translate all the non-ascii characters into amslatex codes.